config/eslint/.eslintrc-source

```diff
@@ -7,3 +7,4 @@ rules:
   # Don't check victory deps until bugfix for NODE_PATH uptream.
   # https://github.com/benmosher/eslint-plugin-import/issues/142
   import/no-unresolved: [2, { ignore: ["^(builder-|)victory"] }]
+  no-magic-numbers: ["error", { "ignore": [-1, 0, 1, 2] }]
```

Comment:
Looks like a great default set!

NIT: Remove quotes around `error` and `ignore`. They are unnecessary.